spec/models/manageiq/providers/embedded_ansible/provider_spec.rb

```diff
@@ -35,5 +35,17 @@
         end
       end
     end
+
+    context "Embedded Ansible role" do
+      let(:miq_server) { FactoryGirl.create(:miq_server) }
```

Comment:
`let` is lazy evaluated so this won't actually get run.

I think a better option would be to do this in a `before` and actually set up the role on that server for the success case.